docs/contribute/design_axioms.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Design Axioms
+permalink: /contributing/design-axioms
+---
+
+## 💚 100% Kotlin
+
+Kotlin is the language of choice for components we develop and is the recommended language for application projects integrating components. We make use of Kotlin idioms and official Kotlin extension libraries (e.g. Coroutines).
+
+Java interoperability is not a goal of this project. We may add annotations for Java interoperability (e.g. `@JvmStatic`) at the request of a consumer to facilitate integration, but we are not considering replacing Kotlin idioms for the sake of operability.
+
+## 🔄 Third-party dependencies
+
+As providers of a framework we strive to not depend on other third-party libraries as long as not strictly needed (e.g. Android Jetpack libraries).
```

Comment:
nit: We strive to be independent of other third-party libraries unless strictly needed...

Reply:
nit: As providers of a framework, we strive [...]